data-syncs/building-data-syncs/README.md

```diff
@@ -1,161 +1,138 @@
-# Build data syncs
+# Build data syncs in Cinchy
 
-## Guide
+Data syncs in Cinchy synchronizes information between different systems. This guide walks you through the process of setting up your data syncs.
```

Comment:
A data sync in Cinchy synchronizes information
or
Data syncs in Cinchy synchronize information

Reply:
A data sync in Cinchy synchronizes information between a source and a target.

We currently don't natively support 2-way syncs, so what you have here could be misunderstood.